src/test/java/org/gridsuite/study/server/NetworkModificationTest.java

```diff
@@ -589,6 +589,95 @@ public void testNetworkModificationEquipment() throws Exception {
         wireMockUtils.verifyNetworkModificationPostWithVariant(stubId, bodyJson, NETWORK_UUID_STRING, VARIANT_ID_2);
     }
 
+    @Test
+    public void testCreateBattery() throws Exception {
+        String userId = "userId";
+        StudyEntity studyEntity = insertDummyStudy(UUID.fromString(NETWORK_UUID_STRING), CASE_UUID, "UCTE");
```

Comment:
UUID studyNameUserIdUuid = insertDummyStudy(UUID.fromString(NETWORK_UUID_STRING), CASE_UUID, "UCTE").getId();

(no need of studyEntity elsewhere)